### PR TITLE
Created Main Mobile version and About Me page

### PR DESCRIPTION
**Pull request by myelin0**
# Changes in this branch pull request:
- Main Works sections for mobile version
- Add About Me page 
- Add styling with grid and flexbox
- fix Linters

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -94,12 +112,205 @@ p {
...
+  padding: 4px 12px;
+  font-size: 12px;
+}
+
+.card-p {
+  margin-top: 12px;
+  margin-left: 0;
+}
+
+.big-btn {
```
Comment (rustam-tolipov) [inline style.css:222]: - [x] Its first position looks great. But, As the requirement says: The button (e.g., "See project") is a link or button tag, and follow the button styling: Active, Hover, and Pressed (see design). When I try to hover your button it is not changing the color.


Your button should follow this design:
Screen Shot 2021-11-27 at 11 51 00 PM

File: style.css [hunk 2]
```diff
@@ -94,12 +112,205 @@ p {
   height: 20px;
 }
 
-.social li:hover,
-.menu-button:hover {
-  background-color: rgb(63, 16, 192);
```
Comment (rustam-tolipov) [inline style.css:99]: - [x] To keep your code clean I would highly suggest you, remove unused commits:


Screen Shot 2021-11-27 at 11 52 52 PM

File: style.css [hunk 3]
```diff
@@ -94,12 +112,205 @@ p {
...
+  list-style-type: disc;
+}
+
+.social-icons {
+  margin: 0;
+}
+
+.about-btn {
+  width: auto;
+}
```
Comment (rustam-tolipov) [inline style.css:302]: - [x] The same technique... You need to use it in here as well... The button (e.g., "See project") is a link or button tag, and follow the button styling: Active, Hover, and Pressed (see design). When I try to hover your button it is not changing the color.

Your button should follow this design:
Screen Shot 2021-11-27 at 11 51 00 PM

File: style.css [hunk 4]
```diff
@@ -1,17 +1,26 @@
 @import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&family=Space+Mono&display=swap');
 
+* {
+  padding: 0;
+}
```
Comment (rustam-tolipov) [inline style.css]: - [x] When I look at your design it looks great to me. However, there is some extra spacing around it. 
As you can see on this image:

Screen Shot 2021-11-28 at 12 02 22 AM
It should be full-filled, with no spacing around it. For fixing this issue you are free to accept my suggestions...


```suggestion
* {
  padding: 0;
  margin:0;
}
```

File: style.css [hunk 5]
```diff
@@ -1,17 +1,26 @@
...
-  -webkit-overflow-scrolling: touch;
+  background-color: #fff;
+  border-bottom-left-radius: 60px;
+  padding-bottom: 64px;
 }
 
 .nav-bar {
-  background-color: transparent;
+  background-color: white;
   margin-top: 44px;
```
Comment (rustam-tolipov) [inline style.css]: This suggestion is for avoiding extra spacing in the first section of this page. You don't need margin-top here.
```suggestion
```

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hi @myelin0 , 

Good job so far!

### HIGHLIGHTS!

- No linters errors.💯

There are some issues that you still need to work on to go to the next project but you are almost there!

jimmy so close

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](<https://microverse.zendesk.com/hc/en-us/articles/1500004088561>) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](<https://airtable.com/shrQAqnBwek5a0O0s>)._

File: style.css (hunk 5 above)
Comment (myelin0) [reply style.css]: Made all changes.Thank you for your review!

**Review by rustam-tolipov — APPROVED** [review]
## Hi @myelin0   👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

image

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**